### ISSUE TITLE
Unable to run Swype on Android 14

### ISSUE THREAD

**Issue by fredgolightly** [DroidFreak32/swype#2]
I realise this is off topic for this but it seems you know your way around Swype. I've been using it for a decade and having updated to Android 14 I can no longer run Swype. Nothing comes close, not Gboard, Yandex, Swiftkey etc.

Are you able to get Swype working on Android 14 at all?

I'd be incredibly grateful if so, as would thousands of others!

**Comment by DroidFreak32** [comment]
Nope, I don't have an A14 device and to make matters worse, moving on there will be no easy way to run 32-bit apps like Swype on phones that drop support for 32bit entirely (pixel 8 etc). While some solutions may work, these are paid and can only be tried for a few hours for free, for ex: https://www.amanieusystems.com/technology

For now you might be able to try installing Swype using:

adb install --bypass-low-target-sdk-block FILENAME.apk

https://developer.android.com/about/versions/14/behavior-changes-all#security

**Comment by Kou-ou-yoku** [comment]
I would be happy to finance your journey to taking swype to 64 bit.

**Comment by ThreeDeeJay** [comment]
For anyone wondering, thankfully Swype is still working fine on Android 16 (LineageOS 23). 👀👌

**Comment by DroidFreak32** [comment]
@ThreeDeeJay likely you are running on a device that still supports 32bit libs.

@Kou-ou-yoku sorry, that is not in my expertise to explore

**Comment by ThreeDeeJay** [comment]
Oh I see. I thought this was more like an OS/stock limitation. Kinda like they're trying to block sideloading or something like that next year.
